src/plugins/opensearch_dashboards_react/public/overview_page/overview_page_header/overview_page_header.test.tsx

```diff
@@ -31,95 +31,265 @@
 import React from 'react';
 import { OverviewPageHeader } from './overview_page_header';
 import { shallowWithIntl } from 'test_utils/enzyme_helpers';
+import { useOpenSearchDashboards } from '../../context';
+import { getLogosMock } from '../../../../../core/common/mocks';
 
-jest.mock('../../app_links', () => ({
-  RedirectAppLinks: jest.fn((element: JSX.Element) => element),
-}));
-
-jest.mock('../../context', () => ({
-  useOpenSearchDashboards: jest.fn().mockReturnValue({
-    services: {
-      application: { capabilities: { navLinks: { management: true, dev_tools: true } } },
-      notifications: { toast: { addSuccess: jest.fn() } },
-    },
-  }),
-}));
-
-afterAll(() => jest.clearAllMocks());
+jest.mock('../../context', () => ({ useOpenSearchDashboards: jest.fn() }));
 
 const mockTitle = 'Page Title';
 const addBasePathMock = jest.fn((path: string) => (path ? path : 'path'));
 
+const mockProps = () => ({
+  addBasePath: addBasePathMock,
+  title: mockTitle,
+  branding: {},
+  logos: getLogosMock.default,
+});
+
 describe('OverviewPageHeader ', () => {
-  describe('in default mode ', () => {
-    test('render logo as custom default mode logo', () => {
-      const branding = {
-        darkMode: false,
-        mark: {
-          defaultUrl: '/defaultModeLogo',
-          darkModeUrl: '/darkModeLogo',
-        },
-      };
-
-      const component = shallowWithIntl(
-        <OverviewPageHeader addBasePath={addBasePathMock} title={mockTitle} branding={branding} />
-      );
-      expect(component).toMatchSnapshot();
-    });
+  beforeAll(() => {
+    // @ts-ignore
+    useOpenSearchDashboards.mockImplementation(() => ({
+      services: {
+        application: { capabilities: { navLinks: { management: true, dev_tools: true } } },
+        notifications: { toast: { addSuccess: jest.fn() } },
+      },
+    }));
+  });
 
-    test('render logo as original default mode opensearch mark', () => {
-      const branding = {
-        darkMode: false,
-        mark: {},
-      };
+  afterAll(() => {
+    jest.clearAllMocks();
+  });
 
-      const component = shallowWithIntl(
-        <OverviewPageHeader addBasePath={addBasePathMock} title={mockTitle} branding={branding} />
-      );
-      expect(component).toMatchSnapshot();
-    });
+  it('renders without overlap by default', () => {
+    const props = {
+      ...mockProps(),
+    };
```

Comment:
Unnecessary memory io?

```suggestion
    const props = mockProps();
```